### PR TITLE
Updating terminology for nodes

### PR DESCRIPTION
**Pull request by mburke5678**

### CODE REVIEW

**Review by codyhoag — COMMENTED** [review]
Just a few suggestions; everything else LGTM!

**Comment by codyhoag** [comment]
Also be sure to update the labels for this before merging. :slightly_smiling_face:

File: modules/nodes-containers-volumes-adding.adoc [hunk 1]
```diff
@@ -75,7 +75,7 @@ values: `json`, `yaml`.
 
 For example:
 
-* To add a new volume source *emptyDir* to DeploymentConfig *registry*:
+* To add a new volume source *emptyDir* to the deployment config *registry*:
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply modules/nodes-containers-volumes-adding.adoc:78]: @codyhoag I fear changing the meaning of the step. I would prefer to leave it as a DC.

File: modules/nodes-nodes-managing-about.adoc [hunk 2]
```diff
@@ -5,14 +5,14 @@
 [id="nodes-nodes-managing-about_{context}"]
 = Modifying Nodes
 
-To make configuration changes to a cluster, or MachinePool, you must create a Custom Resource Definition, or KubeletConfig instance. {product-title} uses the Machine Config Controller to watch for changes introduced through the CRD applies the changes to the cluster.
+To make configuration changes to a cluster, or machine pool, you must create a custom resource Definition, or `KubeletConfig` object. {product-title} uses the Machine Config Controller to watch for changes introduced through the CRD applies the changes to the cluster.
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply modules/nodes-nodes-managing-about.adoc:8]: s/custom resource Definition/custom resource definition

**Comment by mburke5678** [comment]
@codyhoag Thanks for the review. I intended to pull it back and give it another look. But you beat me to it!

File: modules/nodes-containers-volumes-adding.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (codyhoag) [reply modules/nodes-containers-volumes-adding.adoc]: Sure; makes sense. Just wanted to bring up the glossary update; when referring to *deployment config* in general terms, it should be updated to *deployment*. When the specific reference is needed, keeping *deployment config* is fine.

**Pull request merged**